packages/react-vapor/src/components/table-hoc/reducers/TableRowReducers.ts

```diff
@@ -69,6 +69,11 @@ const toggleCollasibleTableRowReducer = (
     return state;
 };
 
+const closeAllCollapsibleRowReducer = (
+    _state: HOCTableRowState[],
+    _action: IReduxAction<unknown>
+): ITableRowToggleCollapsiblePayload[] => [];

```

Comment:
I think the reducer should only change the `opened` field to `false` and not touch the rest of the state.